packages/types/src/index.ts

```diff
@@ -127,8 +149,8 @@ export class EN10168SchemaCertificate {
   @IsNotEmptyObject()
   ProductDescription: Record<string, any>;
 
-  @IsNotEmptyObject()
-  Inspection: Record<string, any>;
+  @isNotEmptyArrayOrObject()
+  Inspection: Record<string, any> | Record<string, any>[];
```

Comment:
That's now inconsistent with the schema as the inspection is not required anymore, `@IsOptional` decorator will fix that.

Reply:
I saw your comment on the schema repo - https://github.com/thematerials-network/EN10168-schemas/pull/38#discussion_r927627271
Should I wait until Hannes confirms, or make the change now?